### PR TITLE
Add screenshots data explorer

### LINKED ISSUE #496: Add screenshots to the data explorer documentation
It would be great to visually show what users can expect.

### PR DESCRIPTION
**Pull request by mrchtr**
fix #496

### CODE REVIEW

**Review by PhilippeMoussalli — APPROVED** [review]
Nice! Thanks Matthias

File: docs/data_explorer.md [hunk 1]
```diff
@@ -24,13 +24,20 @@ The user can specify a pipeline and a specific pipeline run and component to exp
 
 The chosen subset (and the columns within the subset) can be explored in 3 tabs.
 
+![data explorer](https://github.com/ml6team/fondant/blob/main/docs/art/data_explorer/data_explorers.png?raw=true)
```
Comment (RobbeSneyders) [inline docs/data_explorer.md:27]:
```suggestion
![data explorer](https://github.com/ml6team/fondant/blob/main/docs/art/data_explorer/data_explorer.png?raw=true)
```

**Review by RobbeSneyders — CHANGES_REQUESTED** [review]
Thanks @mrchtr!

One small typo in one of the urls. Also, I think we can use relative urls here, which should be a bit more robust.

Can you also move the `Data Explorer UI` section to the top and the `How to use?` to the bottom?

**Comment by mrchtr** [comment]
> Also, I think we can use relative urls here, which should be a bit more robust.

Not sure if this will work. The images where broken some weeks ago. Then we started to replace the relative links with the urls to the images.

**Comment by RobbeSneyders** [comment]
> Not sure if this will work. The images where broken some weeks ago. Then we started to replace the relative links with the urls to the images.

This should only be the case on the main page, since we insert the README there and the urls need to be relevant to different directories on the site vs Github.

**Review by RobbeSneyders — APPROVED** [review]

**Pull request merged**